app/facades/menu/show_facade.rb

```diff
@@ -8,10 +8,18 @@ def new_order
       @order = @order.decorate
     end
 
+    def placed_order?(current_user)
+      menu.orders.find_by(user: current_user)
```

Comment:
Why not `exists?(user: current_user)`
Also, predicate methods should return `true` or `false`, `nil` isn't a valid return value therefore current implementation via `find_by` isn't 100% valid predicate method 😛